js/about/certerror.js

```diff
@@ -49,12 +63,30 @@ class CertErrorPage extends React.Component {
         <span data-l10n-id='certErrorText'></span>&nbsp;
         <span className='errorUrl'>{this.state.url || ''}</span>
         <span className='errorText'>{this.state.error || ''}</span>
+        {this.state.cert
+          ? (<div>
+            <span className='certErrorText'>{'Issued To'}</span>
+            <span className='errorText'>{'Common Name (CN): '}</span>
+            <span className='errorText'>{'Organization (O): '}</span>
+            <span className='errorText'>{'Organization Unit(OU): '}</span>
+            <span className='errorText'>{'Serial Number: '}</span>
+            <span className='certErrorText'>{'Issued By'}</span>
+            <span className='errorText'>{'Common Name (CN): ' + this.state.cert.issuerName}</span>
```

Comment:
I just verified that although sending the entire cert object via IPC causes a renderer crash, sending only the `issuerName` works fine. So maybe to avoid the crash, just display the issuerName in the UI and provide a link for the cert data to be saved directly as a file.